lib/cli.rb

```diff
@@ -4,11 +4,12 @@
 require 'commander'
 require 'tty-exit'
 
-require 'adapters/session_adapter'
-require 'adapters/cry_adapter'
-require 'adapters/ose_adapter'
-require 'models/account'
-require 'models/ose_secret'
+require_relative 'adapters/session_adapter'
+require_relative 'adapters/cry_adapter'
+require_relative 'adapters/ose_adapter'
+require_relative 'models/account'
+require_relative 'models/team'
+require_relative 'models/ose_secret'
```

Comment:
https://stackoverflow.com/questions/735073/best-way-to-require-all-files-from-a-directory-in-ruby ?